range_min : 0.0
range_max : 40.0
angle_min : -180.0
angle_max : 180.0
height_min : -0.3
height_max : 0.5
robot_x_max : 0.3
robot_x_min : -0.3
robot_y_max : 0.1
robot_y_min : -0.1
robot_z_max : 1.0
robot_z_min : -1.5